files/jquery.lazy/update.json

```diff
@@ -3,6 +3,9 @@
   "name": "jquery.lazy",
   "repo": "eisbehr-/jquery.lazy",
   "files": {
-    "include": ["jquery.lazy.min.js"]
+    "include": [
+      "./jquery*.min.js",
```

Comment:
This line was correct, I only said to add ./ to the second line. 
Also no need to open a new PR, just update this one